src/grpc/grpc_server.h

```diff
@@ -66,13 +66,16 @@ struct SslOptions {
 };
 
 // GRPC KeepAlive: https://grpc.github.io/grpc/cpp/md_doc_keepalive.html
+// https://grpc.io/docs/guides/keepalive/
 struct KeepAliveOptions {
   int keepalive_time_ms_{7200000};
   int keepalive_timeout_ms_{20000};
   bool keepalive_permit_without_calls_{false};
   int http2_max_pings_without_data_{2};
   int http2_min_recv_ping_interval_without_data_ms_{300000};
   int http2_max_ping_strikes_{2};
+  int max_connection_age_ms_{INT_MAX};
+  int max_connection_age_grace_ms_{INT_MAX};
```

Comment:
actually per convention can you just set this to 0?

Reply:
If I understand the code correctly, the value inside {} is the default value of the option if the flag is not set. So I want the default values here to be consistent with GRPC default values, see https://grpc.io/docs/guides/keepalive/#keepalive-configuration-specification. 

Basically GRPC by default tend to re-use existing connections as possible as it can to reduce network overhead, I think we should preserve this behavior in Triton unless the flags are set.

Please let me know if my understanding is incorrect.

Reply:
Sorry for the confusion here. I was trying to say instead of specifying the max connection age, just don't specify `max_connection_age_ms_` or `max_connection_age_grace_ms_` upon initialization of the `KeepAliveOptions` struct and never pass these parameters to the server if the values are 0.

Reply:
Thanks for the explanation, I have updated my code change according to comments